src/Data/Swagger.hs

```diff
@@ -112,6 +113,7 @@ module Data.Swagger (
 ) where
 
 import Data.Swagger.Lens
+import Data.Swagger.Optics
```

Comment:
is there some actual symbols, or is it all just instances?

Reply:
Just instances. I added `()` (too bad swagger2 doesn't enable -Wall by default).